src/components/PadSection.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import Pad from './Pad'
+
+PadSection.propTypes = {
+  pads: PropTypes.array.isRequired,
+}
+
+export default function PadSection({ pads }) {
+  return <PadSectionStyled>{pads.map(pad => addPad(pad))}</PadSectionStyled>
+
+  function addPad(pad) {
```

Comment:
Does this function really imrove readabilitiy? Maybe refactor the whole mapping to a new function?